lib/src/interfaces/Layout.ts

```diff
@@ -113,15 +113,51 @@ export interface LayoutSplitView {
   options?: OptionsSplitView;
 }
 
+export interface TopTabs {
+  /**
+   * Set ID of the stack so you can use Navigation.mergeOptions to
+   * update options
+   */
+  id?: string;
+  /**
+   * Set the children screens
+   */
+  children?: any[];
+  /**
+   * Configure top tabs
+   */
+  options?: Options;
+}
+
 export interface LayoutRoot {
   /**
    * Set the root
    */
-  root?: Layout;
+  root: Layout;
   modals?: any;
   overlays?: any;
 }
 
+export interface ExternalComponent {
+  /**
+   * Set ID of the stack so you can use Navigation.mergeOptions to
+   * update options
+   */
+  id?: string;
```

Comment:
Seems like these documentation comments are not relevant

Reply:
So what do you want to me do? :D